static/js/apps/nl_interface/debug_info.tsx

```diff
@@ -279,6 +283,17 @@ export function DebugInfo(props: DebugInfoProps): JSX.Element {
               {debugInfo.mainPlaceDCID})
             </Col>
           </Row>
+          <Row>
+            <b>Entity Detection:</b>
+          </Row>
+          <Row>
+            <Col>
+              Entities Detected: {debugInfo.entitiesDetected.join(", ")}
+            </Col>
+          </Row>
+          <Row>
+            <Col>Entities Resolved: {debugInfo.entitiesResolved}</Col>
+          </Row>
```

Comment:
Could you attach this screenshot to desc?

Reply:
done